assemblers/tcp_reader.go

```diff
@@ -77,7 +77,7 @@ func (reader *tcpReader) reassembledSG(sg reassembly.ScatterGather, ac reassembl
 			req.Body.Close()
 		}
 
-		if entry, matchFound := reader.matcher.GetOrStoreRequest(requestId, ctx.CaptureInfo.Timestamp, req); matchFound {
+		if entry, matchFound := reader.matcher.GetOrStoreRequest(requestId, ctx.CaptureInfo.Timestamp, req, sg.Stats().Packets); matchFound {
```

Comment:
What do you think of moving the packet count into a variable above so it's a little clearer what `sg.Stat().Packets` is?

```suggestion
		// get the number of packets that made up this request
		packetCount := sg.Stats().Packets
		if entry, matchFound := reader.matcher.GetOrStoreRequest(requestId, ctx.CaptureInfo.Timestamp, req, packetCount); matchFound {
```